Sources/FirebaseFirestore/FirestoreErrorCode.swift

```diff
@@ -1,14 +1,34 @@
 // SPDX-License-Identifier: BSD-3-Clause
 
-public struct FirestoreErrorCode: Error {
-  public typealias Code = firebase.firestore.Error
+@_exported
+import firebase
+@_spi(FirebaseInternal)
+import FirebaseCore
 
-  public let code: Code
+public struct FirestoreErrorCode: RawRepresentable, Error {
+  public typealias RawValue = Int
+
+  public let rawValue: Int
   public let localizedDescription: String
 
+  public init(rawValue: Int) {
```

Comment:
Similar to `AuthError` - comment?

Reply:
Resolved by moving to a separate `extensions`